java-squid/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -767,8 +844,13 @@ private static String syntaxNodeToDebugString(Tree syntaxNode) {
       case INT_LITERAL:
         sb.append(' ').append(((LiteralTree) syntaxNode).token().text());
         break;
-      default:
-        //no need to debug other syntaxNodes
+      case METHOD_INVOCATION:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1151)